packages/sdk/src/types/Customers.ts

```diff
@@ -136,3 +137,55 @@ export type CustomersUpdateParams = CustomerRequest & IdOrSourceId
 export type CustomersUpdateResponse = CustomerObject | CustomerUnconfirmed
 
 export type CustomersUpdateConsentsBody = Record<string, boolean>
+
+// New types following guideline
+
+// 0-level types
```

Comment:
`CustomerBase` and `CustomerAddress` are Domain Types
0-level types are `CustomersUpdateInBulkRequestBody` , `CustomersUpdateMetadataInBulkRequestBody` and `CustomersDeletePermanentlyResponseBody`